l10n-screenshots.sh

```diff
@@ -25,13 +25,14 @@ fi
 mkdir -p l10n-screenshots
 
 if [ "$1" = '--test-without-building' ]; then
-  EXTRA_FAST_LANE_ARGS='--test_without_building'
-  shift
+    EXTRA_FAST_LANE_ARGS='--test_without_building'
+    shift
 fi
 
 LOCALES=$*
 if [ $# -eq 0 ]; then
-  LOCALES=$(cat "$PROJECT_DIR/l10n-screenshots-locales.txt")
```

Comment:
It's not easy to parse a Yaml file in bash and I don't think we want to run all locales by default anymore. So I updated the script to force users to tell which locales they want to run. How does this sound to you @isabelrios?

Reply:
sounds good, that would only affect when running them locally, right? on CI, this is passed each time if  I got this right

Reply:
That's right!